lib/gis/proj3.c

```diff
@@ -11,6 +11,9 @@
   \author Original author CERL
  */
 
+/* TODO: the G_database_*() functions should be renamed to G_location_*()
```

Comment:
This should happen before G8 or will have to wait till G9.

Reply:
Renaming the `G_database_*()` functions to `G_location_*()` should be a separate PR, this is only cosmetics to clarify function names.
This PR adds new functionality and needs to be merged before the GRASS reprojection system can be changed to make use of this new functionality.

Reply:
I agree. Just a heads-up to be clear on deadlines.